pkg/storage/driver/secrets.go

```diff
@@ -161,6 +161,7 @@ func (secrets *Secrets) Create(key string, rls *rspb.Release) error {
 	// push the secret object out into the kubiverse
 	if _, err := secrets.impl.Create(context.Background(), obj, metav1.CreateOptions{}); err != nil {
 		if apierrors.IsAlreadyExists(err) {
+			secrets.Log("create: release %q already exists in secrets", rls.Name)
 			return ErrReleaseExists
 		}
 
```

Comment:
Should we have an else here to log a potential unknown other cases?

Reply:
otherwise it returns an error so I think there is no need to log it https://github.com/helm/helm/pull/13104/files#diff-1007a5ca11ecfdc4a0ffaa5b178ad1c03d1a8e1cc625288656b1e5ab25765382R168